src/kubeconfig.ts

```diff
@@ -39,12 +40,13 @@ export function readKubectlConfig() : Promise<KubeConfig> {
             const currentUser = currentUserDef['user'];
             const clientCertData = currentUser['client-certificate-data'];
             const clientKeyData = currentUser['client-key-data'];
-
+            const clientCertDataFile = currentUser['client-certificate'];
+            const clientKeyFile = currentUser['client-key'];
             resolve({
                 endpoint: endpoint,
-                clientCertificateData: Buffer.from(clientCertData, 'base64'),
-                clientKeyData: Buffer.from(clientKeyData, 'base64'),
-                certificateAuthorityData: Buffer.from(cadata, 'base64')
+                clientCertificateData: clientCertDataFile ? <Buffer>fs.readFileSync(clientCertDataFile): Buffer.from(clientCertData, 'base64'),
```

Comment:
TS docs seem to now prefer `foo as Buffer` to `<Buffer>foo`, though if we have two distinct members in the interface then we can get rid of the type assertion altogether.